src/MSALWrapper/TokenResult.cs

```diff
@@ -25,6 +25,11 @@ public enum AuthType
         /// Device code flow auth type.
         /// </summary>
         DeviceCodeFlow,
+
+        /// <summary>
+        /// Integrated Windows auth flow auth type.
+        /// </summary>
+        IntegratedWindowsAuthenticationFlow,

```

Comment:
@mvanchaa - Heads Up I expect your upcoming telemetry work to collide with this auth_type change (maybe not as an actual merge conflict but as something to reconcile). 

Right now we had Silent, Interactive, and DeviceCodeFlow - however, DeviceCodeFlow could technically be Silent.  SO I expect we want to drop Silent/Interactive here and move to having each authFlow have an interactive bool field. We will know which flow it was from the event itself, and then this auth_type field can be deprecated (Since the type will be dynamically computed from the AuthFlow class in use).